defaults.env

```diff
@@ -108,3 +108,6 @@ PROJECT_REVIEW_TOKEN=
 RAISE_SIGNIN_NOTIFICATIONS=no
 
 LOAD_BRAINTREE_FORM_ON_HOMEPAGE=no
+
+LIBERAPAY_BASE_URL=http://localhost:8339
+LIBERAPAY_SECRET=fake
```

Comment:
Do I need something here to test? Or do I need Liberapay running locally too?

Reply:
If you want to test the full migration process then yes, you need both Gratipay and Liberapay running locally.

Reply:
Thanks! Working on it...